torch_lr_finder/lr_finder.py

```diff
@@ -6,6 +6,16 @@
 from torch.optim.lr_scheduler import _LRScheduler
 import matplotlib.pyplot as plt
 
+try:
+    from apex import amp
+    IS_AMP_AVAILABLE = True
+except ImportError:
+    import logging
+    logger = logging.getLogger(__name__)
+    logger.warning('To enable mixed precision training, please install `apex`.')
```

Comment:
I would add the link to the apex repository or the command to reinstall this package with the amp option: `pip install torch-lr-finder -v --global-option="amp"`

Reply:
Ok, I'll add the installation command in the message!